src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -145,9 +144,9 @@ File File::getSpecialLocation (const SpecialLocationType type)
 class DirectoryIterator::NativeIterator::Pimpl
 {
 public:
-    Pimpl (const File& directory, const String& wildCard_)
+    Pimpl (const File& directory, const String& _wildCard)
```

Comment:
There should be no controversy here, its addressed in our coding style:
http://i.imgur.com/Z7MSpl1.png